docs/getting-started-guides/kubeadm.md

```diff
@@ -195,6 +195,38 @@ Once a pod network has been installed, you can confirm that it is working by che
 
 And once the `kube-dns` pod is up and running, you can continue by joining your nodes.
 
+If you see following status 
+```
+NAMESPACE     NAME                              READY     STATUS              RESTARTS   AGE
+kube-system   canal-node-f0lqp                  2/3       RunContainerError   2          48s
+```
+Or 
+
+```
+kube-system   canal-node-77d0h                  2/3       CrashLoopBackOff    3          3m
+kube-system   kube-dns-2924299975-7q1vq         0/4       ContainerCreating   0          15m
+```
+The three status ```RunContainerError``` and ```CrashLoopBackOff``` and ```ContainerCreating``` very common. 
```

Comment:
**Grammar:** 

The three **statuses** `RunContainerError`**,** `CrashLoopBackOff`**,**  and `ContainerCreating` **are** very common.